raiden_libs/messages/json_schema.py

```diff
@@ -145,6 +145,55 @@ def make_properties_required(schema):
     }
 }
 
+PATHS_REPLY_SCHEMA = {
+    'type': 'object',
+    'required': [
+        'token_network_address',
+        'target_address',
+        'value',
+        'num_paths',
+        'chain_id',
+        'nonce',
+        'paths',
+        'estimated_fee',
+        'signature',
+    ],
+    'properties': {
+        'token_network_address': {
+            'type': 'string',
+        },
+        'target_address': {
+            'type': 'string',
+        },
+        'value': {
+            'type': 'integer',
+            'minimum': 1
+        },
+        'num_paths': {
+            'type': 'integer',
+            'minimum': 1
+        },
+        'chain_id': {
+            'type': 'integer',
+            'minimum': 1
+        },
+        'nonce': {
```

Comment:
Do we need a nonce in the answer?

Reply:
i think so yes, replay from adversary to client possible otherwise.

Reply:
But we know when we sent a request. Is that enough?

Reply:
Hmm, maybe this is overenginerring, but if an attacker is able to respond faster than the pfs, the replay would still work.